atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -370,7 +370,7 @@ SerializableTransactionManager serializable() {
                 config.keyValueService().concurrentGetRangesThreadPoolSize(),
                 config.keyValueService().defaultGetRangesConcurrency(),
                 config.initializeAsync(),
-                config.getTimestampCacheSize());
+                () -> runtimeConfigSupplier.get().getTimestampCacheSize());
```

Comment:
probably eventually want JavaSuppliers.compose(), though for now this will do

Reply:
Is that change complicated enough to defer?

Reply:
It turns out we use java suppliers on TM, but guava suppliers everywhere else. Let's keep it like this for now, and change it after we change all our suppliers to Java suppliers.